revscoring/features/util.py

```diff
@@ -0,0 +1,11 @@
+import re
+
+CATEGORY_RE = re.compile(r"^category:", re.U|re.I)
+CITE_RE = re.compile(r"cite", re.I|re.U)
+IMAGE_RE = re.compile(r"^(file|image):", re.U|re.I)
+INFOBOX_RE = re.compile(r"infobox|sidebar", re.I|re.U)
```

Comment:
The regexes CATEGORY_RE, CITE_RE, IMAGE_RE and INFOBOX_RE should be configurable per language (they are English only for now)